index.html

```diff
@@ -142,15 +151,15 @@ <h2 class="heading">About
           <li><a href="#"><img src="images/social/github.svg" alt="github"></a></li>
           <li><a href="#"><img src="images/social/vector.svg" alt="vector"></a></li>
         </ul>
-        <button class="bg round-borders cap resume" type="button">Get my resume</button>
+        <button class="btn bg round-borders cap resume" type="button">Get my resume</button>
       </div>
-
+        
+      <div class="qualifications">
       <details class="ability" open>
-        <summary>
+        <summary >
           <h3> Languages</h3>

```

Comment:
- [ ] Consider applying the correct color for this `h3` element so that it matches your design template. From your design, I could see that this `#344563;` is the color for this `h3` element on the desktop version. Please check your template and make corrections.